src/overloads/hessian_tracer.jl

```diff
@@ -13,22 +13,36 @@ end
 
 function hessian_tracer_1_to_1_inner(
     p::P, is_der1_zero::Bool, is_der2_zero::Bool
-) where {I,SG,SH,P<:IndexSetHessianPattern{I,SG,SH}}
+) where {I,SG,SH,P<:IndexSetHessianPattern{I,SG,SH,false}}
     sg = gradient(p)
     sh = hessian(p)
     sg_out = gradient_tracer_1_to_1_inner(sg, is_der1_zero)
     sh_out = if is_der1_zero && is_der2_zero
         myempty(SH)
     elseif !is_der1_zero && is_der2_zero
         sh
-    elseif is_der1_zero && !is_der2_zero
+    elseif is_der1_zero && is_der2_zero
```

Comment:
missing a `!` here and it wasn't caught by the tests

Reply:
Does this case even exist?

Reply:
yes locally